src/components/dialog/_dialog.scss

```diff
@@ -176,11 +176,11 @@ $dialogMaxWidthMap: (
 
     &--appearance-dialog {
       background-color: $white;
-      border-radius: 24px 24px 0 0;
+      border-radius: var(--border-radius-l) var(--border-radius-l) 0 0;
       box-shadow: $dialogBoxShadow;
 
       @include sgResponsive(md) {
-        border-radius: $borderRadiusDefault;
+        border-radius: var(--border-radius-xs);
```

Comment:
according to Wojtek's documentation border radius for desktop dialog should be `border-radius: var(--border-radius-l);`

https://www.figma.com/file/hJG4ROfLYw6lymxeVuQRyy/%5BDS%5D-Border-Radius?node-id=228-14520&t=rHnBbnCpt0d0GWJY-4

Reply:
done